src/GovUk.Education.ExploreEducationStatistics.Admin/Services/LegacyReleaseService.cs

```diff
@@ -56,16 +67,59 @@ public class LegacyReleaseService : ILegacyReleaseService
                 .OnSuccess(publication =>
                 {
                     return publication.LegacyReleases
-                        .OrderByDescending(legacyRelease => legacyRelease.Order)
                         .Select(legacyRelease =>
                             new LegacyReleaseViewModel
                             {
                                 Id = legacyRelease.Id,
                                 Description = legacyRelease.Description,
-                                Order = legacyRelease.Order,
+                                Order = publication.ReleaseOrders.Find(ro => ro.ReleaseId == legacyRelease.Id)?.Order ?? 0,
                                 Url = legacyRelease.Url
-                            }
-                        ).ToList();
+                            })
+                        .OrderByDescending(lr => lr.Order)
+                        .ToList();
+                });
+        }
+
+        public async Task<Either<ActionResult, List<CombinedReleaseViewModel>>> ListCombinedReleases(Guid publicationId)
+        {
+            return await _persistenceHelper
+                .CheckEntityExists<Publication>(publicationId)
+                .OnSuccess(publication => _userService.CheckCanViewPublication(publication))
+                .OnSuccessCombineWith(async _ => await ListLegacyReleases(publicationId))
+                .OnSuccessCombineWith(async _ => await _publicationService.ListLatestReleaseVersions(publicationId))
+                .OnSuccess(publicationAndReleases =>
+                {
+                    var (publication, legacyReleases, eesReleases) = publicationAndReleases;
+                    var combinedReleasesViewModels = new List<CombinedReleaseViewModel>();
+
+                    foreach (var legacyRelease in legacyReleases)
+                    {
+                        combinedReleasesViewModels.Add(new()
+                        {
+                            Description = legacyRelease.Description,
+                            Id = legacyRelease.Id,
+                            Url = legacyRelease.Url,
+                            Order = legacyRelease.Order,
+                            IsLegacy = true
+                        });
+                    }
+
+                    foreach (var eesRelease in eesReleases)
+                    {
+                        combinedReleasesViewModels.Add(new()
+                        {
+                            Description = eesRelease.Title,
+                            Id = eesRelease.Id,
+                            Url = $"{_options.BaseUrl}/find-statistics/{publication.Slug}/{eesRelease.Slug}",
```

Comment:
The frontend has access to the environment base URL. Search for `.tsx` files containing `config.publicAppUrl` and you should see examples like 
```tsx
href={`${config.PublicAppUrl}/find-statistics/${publicationSlug}/${releaseSlug}`}
```

Personally I don't think you need the link for non-legacy releases and there could be other problems with those links.

-  First version releases that are not published will have a link that's not found.
- Unpublished amendments will have a link that goes to the latest published version.
 
On the 'Releases' tab you won't see links to the public releases either. The standard way to get the public link to a release is to go to the Sign off tab of that release where there's a link under the title `The public release will be accessible at:`...

Reply:
I'll update this so that first version drafts don't show a URL, since it will be broken.

Draft amendments will retain the original release URL as this was the intended behaviour. The draft "overlays" the original release on the reordering page until it is published, so it makes sense for the URL to be for the original release.